source/_lovelace/picture-glance.markdown

```diff
@@ -25,30 +23,80 @@ You can also use `camera` domain entities to use that as image. You can also use
 Screenshot of the picture glance card.
 </p>
 
-**Options**
+{% linkable_title Options %}
 
-| Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
-| type | string | **Required** | `picture-glance`
-| image | string | **Required** | URL of an image
-| entities | list | **Required** | Entity id's
-| navigation_path | string | Optional | Path of URL to use in navigation
-| camera_image | string | Optional | camera domain entity_id 'camera.demo_camera'
-| state_image | object | Optional| See `state_image` object structure.
-| entity | list | Optional | An entity to use for state_image state
-| title | string | Optional | Card title
+{% configuration %}
+type:
+  required: true
+  description: picture-glance
+  type: string
+image:
+  required: true
+  description: URL of an image
+  type: string
+entities:
+  required: false
+  description: Entity id's
+  type: list
+  default: None
+navigation_path:
+  required: false
+  description: Path of URL to use in navigation
+  type: string
+  default: None
+camera_image:
+  required: false
+  description: "Camera domain entity_id 'camera.demo_camera'"
+  type: string
+  default: None  
+state_image:
+  required: false
+  description: "See `state_image` object structure."
+  type: object
+  default: None
+entity:
+  required: false
+  description: An entity to use for state_image state
+  type: list
+  default: None
+title:
+  required: false
+  description: Card title
+  type: string
+  default: None
+{% endconfiguration %}
 
 `state_image` object structure
 
-| Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
-| on | string | Optional | URL of an image used for on state.
-| off | string | Optional | URL of an image used for off state.
-| home | string | Optional | URL of an image used for home state.
-| not_home | string | Optional | URL of an image used for not_home state.
-| ... | string | Optional | Any state that is supported by the entity works
+{% configuration %}
+"on":
+  required: true
+  description: URL of an image used for on state.
+  type: string
+  default: None
+"off":
+  required: true
+  description: URL of an image used for off state.
+  type: string
+  default: None
+home:
+  required: false
+  description: URL of an image used for home state.
+  type: string
+  default: None
```

Comment:
you should check where `default: None` makes sense to mention; elsewhere remove it